content/docs/using-github-directly/index.mdx

```diff
@@ -0,0 +1,303 @@
+
+Now that you have all the packages you need to preview DARIAH-Campus, you can start the server and build the site like this:
+
+1. In Terminal, go to your local DARIAH-Campus folder (see previous steps 1, and 3 in the previous section for how to do this)
+2. Type "`yarn develop`" (+ enter). This build the whole site locally and start a server to run it in. This may also take a little while.
+3. Once the build process is complete, you will be able to view DARIAH-Campus in the browser using the link given to you in the terminal, which will look like this: ⠀ `http://localhost:8000/`
+4. Copy the url given in Terminal into your browser, which will give you the DARIAH-Campus home page. _Any saved edits you make to the DARIAH-Campus .mdx files in your editor will automatically be shown in your browser._
+
+### Instructions for Windows users
+
+1. Download node.js from [https://nodejs.org/en/download/](https://nodejs.org/en/download/)
+2. Run node.js installer
+3. Open 'command line' and check node version by typing: `node -v`
+4. Download yarn from [https://yarnpkg.com/lang/en/docs/install/#windows-stable](https://yarnpkg.com/lang/en/docs/install/#windows-stable)
+5. Type `Run yarn installer` (enter)
+6. Check yarn version: `yarn --version`
+7. Install the necessary pacakges by typing `yarn install` (+ enter)
+8. In GitHub desktop, make sure that you have the most up-to-date version of DARIAH-Campus. To do this, follow the instructions above to 'Fetch Origin' and 'Merge' it to your local branch
+9. Once any changes from the DARIAH-Campus 'master' on GitHub has been merged to your local branch, navigate to inside the dariah-campus directory (where the file package.json is located) in your file manager.
+10. Run: `yarn develop`. This will build the entire website and launch a server to run it in on your computer. Once the build process is complete, you can:
+11. Navigate to `localhost:8000` in your browser to preview DARIAH-Campus. _Any saved edits you make to the DARIAH-Campus .mdx files in your editor will automatically be shown in your browser._
+
+> **REMEMBER!** Any changes you make in Atom will only show up on your local version, but will not make any changes to the actual live version of DARIAH-Campus. This is useful if you want to try out a few things before updating. However, it’s important to remember that if you WANT to make changes to the live version, you will need to ‘commit’ them, ‘push origin’ and then ‘create a pull request’ so that the DARIAH-Campus editors can approve the changes.
+
+
+# Doing the Work
+
+When you work on your own learning resource, you will work locally, on your computer, in the locally cloned fork of DARIAH-Campus.
+
+To help you start working on your contribution in Markdown, we have created templates that you can use.
+
+## Starting with a template
+
+### Hosted resource
+
+Once you have set up DARIAH-Campus on your computer, you can use the `resource-template-folder` as a basis for your contribution.
```

Comment:
do we still want those? if yes i'll need to add them back

Reply:
yes we do, please :)  Also, I noticed there is a resource in the original 'Contents' folder, was this a duplicate?